primitives/npos-elections/src/helpers.rs

```diff
@@ -95,15 +95,15 @@ mod tests {
 			Assignment {
 				who: 1u32,
 				distribution: vec![
-					(10u32, Perbill::from_fraction(0.5)),
-					(20, Perbill::from_fraction(0.5)),
+					(10u32, Perbill::from_float(0.5)),
```

Comment:
Would be good to ensure new names are consistent with `FixedPointNumber`. I think the only one needing revision os this ^^, it is still called `from_fraction` there. Can also be a follow up.

Reply:
@kianenigma please open an issue